apps/nextjs/scripts/increase-listeners.js

```diff
@@ -0,0 +1,7 @@
+// Increase the limit of max listeners
```

Comment:
Allows us to have more listeners since the project has lots of files that are touched each build. Fixes a "too many listeners" error in local dev